### PR TITLE
[Form] Fix `Forms` docblock to use FQCN

### PR DESCRIPTION
**Pull request by HeahDude**
| Q | A |
| --- | --- |
| Branch? | 2.8+ |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by HeahDude** [comment]
Note to merger: line 32 with `choices_as_values` can be removed in 3.0+.

**Comment by phansys** [comment]
Indeed, in 3.0+ the FQCN string representation could be replaced with the `class` constant of each class.

**Comment by HeahDude** [comment]
@phansys It may not be necessary as that would mean to add the corresponding `use` statement...

**Comment by phansys** [comment]
IMO, it would be better to import the classes with `use` instead of hardcoding the FQCN, which could lead to typos and haven't IDE support. Obviously, I'm talking about the real use cases, not about those examples, in which I think you can use `FQCN::class` syntax.

**Comment by HeahDude** [comment]
I'm fine either way :)

**Comment by Tobion** [comment]
Thank you @HeahDude.

File: src/Symfony/Component/Form/Forms.php [hunk 1]
```diff
@@ -24,11 +24,12 @@
  * $formFactory = Forms::createFormFactory();
  *
  * $form = $formFactory->createBuilder()
- *     ->add('firstName', 'text')
- *     ->add('lastName', 'text')
- *     ->add('age', 'integer')
- *     ->add('gender', 'choice', array(
- *         'choices' => array('m' => 'Male', 'f' => 'Female'),
+ *     ->add('firstName', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```
Comment (webmozart) [inline src/Symfony/Component/Form/Forms.php]: I think we should use `::class` constants here. Almost nobody uses PHP < 5.5 nowadays anyway.

Comment (javiereguiluz) [inline src/Symfony/Component/Form/Forms.php]: Sadly that's not true :sob: Worst case: ~66% of WordPress users run PHP 5.4 or lower ([source](https://wordpress.org/about/stats/)). Best case: ~25% of SonataProject users run PHP 5.4 or lower ([source](https://twitter.com/sonataproject/status/699278314817593344)) and ~25% of Packagist/Composer users run PHP 5.4 or lower ([source](https://seld.be/notes/php-versions-stats-2015-edition))

**Pull request merged**